js/ui/toolbar/ui.toolbar.base.js

```diff
@@ -345,7 +355,7 @@ var ToolbarBase = CollectionWidget.inherit({
     },
 
     _renderItems: function(items) {
-        var grouped = items.length && items[0].items;
+        var grouped = this.option("grouped") && items.length && items[0].items;
```

Comment:
You should describe the `dxToolbarItemTemplate.items` field

Reply:
This option works incorrectly with adaptivity. It is the reason to delay publishing before the fix